docs/walk-through/loops.md

```diff
@@ -2,12 +2,13 @@
 
 When writing workflows, it is often very useful to be able to iterate over a set of inputs, as this is how argo-workflows can perform loops.
 
-There are two basic ways of running a template multiple times.
+There are three basic ways of running a template multiple times.
 
 - `withItems` takes a list of things to work on. Either
     - plain, single values, which are then usable in your template as '{{item}}'
     - a JSON object where each element in the object can be addressed by it's key as '{{item.key}}'
 - `withParam` takes a JSON array of items, and iterates over it - again the items can be objects like with `withItems`. This is very powerful, as you can generate the JSON in another step in your workflow, so creating a dynamic workflow.
+- `withSequence` take a Sequence, and convert it into a numeric sequence.
 
 ## `withItems` basic example
```

Comment:
on line 15 below (GitHub won't let me comment on it, unfortunately), `withItems` would no longer be the simplest, so that line should be reworded.
k8s style guide also says to [avoid words that assume a level of understanding](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-words-that-assume-a-specific-level-of-understanding), so it shouldn't say "simplest" at all (nor "basic").

I'd remove the first sentence entirely and simplify the second one (which is not in [present tense](https://kubernetes.io/docs/contribute/style/style-guide/#use-present-tense), [active voice](https://kubernetes.io/docs/contribute/style/style-guide/#use-active-voice) etc)

Reply:
OK, I move it to first

Reply:
This was about a different line, see follow-up comment: https://github.com/argoproj/argo-workflows/pull/12577#discussion_r1498170713